### ISSUE TITLE
[Unblock Legitimate Site]

### ISSUE THREAD

**Issue by Voks344** [MetaMask/eth-phishing-detect#14765]
**Domain:**
http://metamask.github.io/phishing-warning/v3.0.0/service-worker.js.  

**Details:** *Some details about the domain (optional)*
When I open chrome browser to the google site, with metamask extension, my virus scanner instantly blocks this http://metamask.github.io/phishing-warning/v3.0.0/service-worker.js.  

- [ ] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
im not sure what site you are talking about. please open a new issue with the actual site.
